LBHackney-IT/lbh-fss-portal-frontend
# Feature/organisation approve decline delete

**Description**

Allow admin to approve, decline or remove organisation - in each instance a pop-up modal should appear to confirm the action, for approve and decline there should be an optional text box to write a reviewer message 

**Test instructions**

- pull this branch 
- check approve, decline and remove functionality works as expected 
- compare against figma designs 

Figma designs:
https://www.figma.com/file/t6fanDKvLkCHJsY85ivFTF/Services-Directory?node-id=598%3A44959

![image](https://user-images.githubusercontent.com/69238887/93203972-33186280-f74d-11ea-9812-3d6e98a904cb.png)


**Tickets**

https://app.clickup.com/t/9e5u8z
https://app.clickup.com/t/7ev9uq
